target/linux/mediatek/dts/mt7981b-openfi-6c.dts

```diff
@@ -0,0 +1,264 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+#include "mt7981b.dtsi"
+
+/ {
+	model = "OpenFi 6C";
+	compatible = "openfi,6c", "mediatek,mt7981";
```

Comment:
Please use `mediatek,mt7981b` instead of `mediatek,mt7981`.

https://elixir.bootlin.com/linux/v6.12.39/source/Documentation/devicetree/bindings/arm/mediatek.yaml#L91